config/crd/bases/xgboostjob.kubeflow.org_xgboostjobs.yaml

```diff
@@ -8,6 +8,19 @@ metadata:
   creationTimestamp: null
   name: xgboostjobs.xgboostjob.kubeflow.org
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.conditions[-1:].type
+    name: State
+    type: string
+  - JSONPath: .metadata.creationTimestamp
+    name: Age
+    type: date
```

Comment:
creationTimeStamp is not like age, age should be currentTime - creationTime

Reply:
the JSONPath seems do not support expressions, and `.metadata.creationTimestamp` works well, outputs of kubectl shows the duration as `currentTime - creationTime`.